src/NUnitConsole/nunit3-console/ResultReporter.cs

```diff
@@ -42,6 +42,8 @@ public ResultReporter(XmlNode resultNode, ExtendedTextWriter writer, ConsoleOpti
             OverallResult = resultNode.GetAttribute("result");
             if (OverallResult == "Skipped")
                 OverallResult = "Warning";
+            if (OverallResult == null)
+                OverallResult = "Unknown";
```

Comment:
Seems simple enough. Does it get colored red, however? Does any error show up in the console report?